src/Templating/EngineInterface.php

```diff
@@ -13,8 +13,55 @@
 
 namespace Sonata\BlockBundle\Templating;
 
-use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface as TemplatingEngineInterface;
+use Symfony\Component\HttpFoundation\Response;
 
-interface EngineInterface extends TemplatingEngineInterface
+/**
+ * @deprecated since 4.0, to be removed with 5.0.
```

Comment:
If not used, you can remove it also

Reply:
It is not used =)
If so, i will remove templating helper too! All methods will be copied to twig extension